packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -146,6 +146,7 @@ Widget _wrapWithBackground({
   Brightness? brightness,
   required Widget child,
   bool updateSystemUiOverlay = true,
+  bool disableBackgroundFilterBlur = false,
```

Comment:
Rename to `enableBackgroundFilterBlur`: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-double-negatives-in-apis